.github/workflows/main.yaml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   make-unit-tests:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
How about pinning it to 24.04 to reduce random breakages when upstream is updated?